examples/twilio-sms-connector/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <artifactId>twilio-demo-connector</artifactId>
+  <groupId>com.salesforce.einsteinbot.twilio</groupId>
+  <modelVersion>4.0.0</modelVersion>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <groupId>org.springframework.boot</groupId>
+        <version>${spring-boot-maven-plugin.version}</version>
+      </plugin>
+    </plugins>
+  </build>
+  <dependencies>
+
+    <dependency>
+      <artifactId>einstein-bot-channel-connector-java-starter</artifactId>
+      <groupId>com.salesforce.einsteinbot</groupId>
+      <version>${channel-connector-java-starter.version}</version>
+    </dependency>
+
+    <dependency>
+      <artifactId>twilio</artifactId>
+      <groupId>com.twilio.sdk</groupId>
+      <version>${twilio-sdk.version}</version>
+    </dependency>
+
+
+  </dependencies>
+
+  <name>twilio-connector</name>
+
+  <parent>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <groupId>org.springframework.boot</groupId>
+    <version>2.6.6</version> <!-- lookup parent from repository -->
+  </parent>
+
+  <properties>
+    <bootstrap.version>4.2.1</bootstrap.version>
+    <java.version>1.8</java.version>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <spring-boot-maven-plugin.version>2.4.1</spring-boot-maven-plugin.version>
+    <channel-connector-java-starter.version>2.0.2</channel-connector-java-starter.version>
+    <twilio-sdk.version>7.47.2</twilio-sdk.version>
+  </properties>
+
+  <version>1.0.0-SNAPSHOT</version>
```

Comment:
Lets have the same version as the parent:
https://github.com/forcedotcom/einstein-bot-channel-connector/blob/dabfbe210bace12680ab1a4249f54d493b2535a5/examples/twitter-connector/pom.xml#L13